bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/config/impl/HttpCacheConfigImpl.java

```diff
@@ -111,14 +111,6 @@
                name = "cacheConfigExtension")
     private HttpCacheConfigExtension cacheConfigExtension;
 
-    // Custom cache config attributes
```

Comment:
Removed customAttribures as this is the domain of the custom extension. Some other extension may need 2 lists and a checkbox... no one knows!